linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_command/rule.yml

```diff
@@ -33,30 +33,20 @@ references:
     stigid@rhel8: RHEL-08-020040
     vmmsrg: SRG-OS-000028-VMM-000090,SRG-OS-000030-VMM-000110
 
-ocil_clause: 'lock-command is not set'
+ocil_clause: 'the "lock-command" is not set in the global settings to call "vlock"'
 
 ocil: |-
-    To verify that vlock is configured as a locking mechanism in tmux, run the following command:
+    Verify {{{ full_name }}} enables the user to initiate a session lock with the following command:
 
-    <pre>$ sudo grep lock-command /etc/tmux.conf</pre>
-
-    The output should return the following:
+    <pre>$ grep lock-command /etc/tmux.conf</pre>
 
     <pre>set -g lock-command vlock</pre>
 
-    Then, verify that the /etc/tmux.conf file can be read by other users than root:
```

Comment:
We have add the file permission checks to the tmux rules by https://github.com/ComplianceAsCode/content/pull/9080 as a solution for https://bugzilla.redhat.com/show_bug.cgi?id=2064696. Back then, we decided (discussion in the PR) to put the file permissions into the rule because they are tightly coupled with the actual config option and it's hard to track dependencies among rules.

Now I don't know, we can have a special field with a check text just for STIG, but then it could be seen as misaligned content.

Reply:
The problem will be tracked in https://github.com/ComplianceAsCode/content/issues/9533